### ISSUE TITLE
Gravity of DigitalTextViews not working (pin input is not being centered in the DigitTextView)

### ISSUE THREAD

**Issue by Jawnnypoo** [Philio/PinEntryView#2]
Something is going wrong in either onMeasure or onLayout of this custom view that is causing the DigitViews to not get their "setGravity(Gravity.CENTER)" set properly. It is honoring the center horizontal, but not the center vertical, which leads me to believe that the error is either with onMeasure or onLayout. I fixed this by converting the custom view to a FrameLayout, which didnt require overriding onMeasure or onLayout, and gave it children of the EditText (which fills the entire parent) and a LinearLayout that is set to horizontal orientation, and adds the children dynamically. I can help with that part of the code if you want, it is a pretty big change though.

**Comment by Philio** [comment]
Thanks for reporting this, I hadn't noticed this as with all the default options it looks nicely centered, however if you play around a bit it's quite obvious that vertical centering isn't working.

The problem is just that the digits do not get measured so a simple fix is just to call measure on the digits from onMeasure in the view group.

I pushed a fix in the code and once I've sorted out a couple of lint errors causing the publishing to fail the artifacts will be available too.
